core/src/datayoga_core/blocks/http/receiver/block.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from abc import ABCMeta
+from asyncio import Queue
+from contextlib import suppress
+from itertools import count
+from typing import AsyncGenerator, List, Optional
+
+import orjson
+from aiohttp.web import BaseRequest, Response, Server, ServerRunner, TCPSite
+from datayoga_core.context import Context
+from datayoga_core.producer import Message
+from datayoga_core.producer import Producer as DyProducer
+
+logger = logging.getLogger("dy")
+
+
+class Block(DyProducer, metaclass=ABCMeta):
+    port: int
+
+    def init(self, context: Optional[Context] = None):
+        logger.debug(f"Initializing {self.get_block_name()}")
+        self.port = int(self.properties.get("encoding", 8080))
+
+    async def produce(self) -> AsyncGenerator[List[Message], None]:
+        q = Queue(maxsize=1000)
+
+        async def handler(request: BaseRequest) -> Response:
+            try:
+                q.put_nowait(orjson.loads(await request.read()))
+                return Response(status=200)
+            except Exception:  # noqa
+                logger.exception("Got exception while parsing request:")
```

Comment:
can you also log the exception itself?

Reply:
The `exception` method is unwinding current stacktrace implicitly. Don't need to pass exception here.